cql3/restrictions/statement_restrictions.hh

```diff
@@ -125,7 +125,8 @@ private:
     /// Recall that the index-table CK is (token, PK, CK) of the base table for a global index and (indexed column,
     /// CK) for a local index.
     ///
-    /// Elements are single-column binary operators.  The first element's RHS is a dummy value.
+    /// Elements are conjuctions of single-column binary operators.
```

Comment:
Should we add a bit more info?  It could be helpful to future readers who need to work with this field.  Eg,

> Elements are conjuctions of single-column binary operators with the same LHS.  Element order follows the indexing-table clustering key.

I guess it depends on how descriptive one finds "ck_prefix" in the name.  It seems obvious to me what it is, but that's not the best guide... :)

Reply:
> Elements are conjuctions of single-column binary operators with the same LHS. Element order follows the indexing-table clustering key.

I added this, but I don't know what else to write here. We can't explain everything that goes on in `prepare_indexed`.